### PR TITLE
Resquashed commit of avatar thumbnail fix.

### LINKED ISSUE #1850: Profile Page shows Active User Avatar for Thumbnail of both Gravatar and Image
This is a bit disconcerting. Whichever Profile image is currently being used (Gravatar or Uploaded Image), the thumbnail of the active image is shown as the thumbnail for both gravatar and uploaded image. Here is a gif to show the issue. (The triangle thingy is my gravatar, the other one is asciiart in retroterm that I uploaded to the local filesystem.) Furthermore, it appears that the string "Use Gravatar" has not been translated into Esperanto or any other language yet...

bildo

### PR DESCRIPTION
**Pull request by denjello**
This attempts to fix #1850

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -175,7 +175,7 @@
     it "returns gravatar url if avatar_kind is 'gravatar'" do
       user.should_receive(:gravatar_url).and_return('www.gravatar/spike')
       user.avatar_kind = 'gravatar'
-      user.avatar_url(:small).should == 'www.gravatar/spike'
+      user.avatar_url(:small,user.avatar_kind).should == 'www.gravatar/spike'
```
Comment (s01us) [inline spec/models/user_spec.rb]: req rspec format is : 


```ruby
expect(user.avatar_url(:small,user.avatar_kind)).to eq 'www.gravatar/spike'
```

**Comment by robguthrie** [comment]
@denjello where do files like /uploaded_avatars/medium/missing.png come from?

**Comment by denjello** [comment]
I believe I just carried that over from existing code.

**Comment by robguthrie** [comment]
So will it do something useful for other Loomio instances? I don't think
the file urls will resolve.

**Comment by denjello** [comment]
No - I didn't add that, it was already in the test spec.

**Comment by denjello** [comment]
This is the original line from the master git repo at loomio/loomio:
 app/models/user.rb:318

` uploaded_avatar.url(:medium) != '/uploaded_avatars/medium/missing.png' `

**Comment by s01us** [comment]
@denjello we made a change to the github history to remove some CRAZY bloat.
I've cherry-picked your commit into a nice clean PullRequest : https://github.com/loomio/loomio/pull/1903 <3

**Pull request closed without merging**